src/Components/People.js

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from "react";
+import axios from "axios";
+
+class People extends Component {
+  constructor() {
+    super();
+    this.state = {
+      searchPerson: "",
+      currentPerson: {},
+      personName: "",
+      isError: false,
+    };
+  }
+
+  handleChange = (e) => {
+    this.setState({
+      searchPerson: e.target.value,
+    });
+  };
+
+  handleSubmit = async (e) => {
+    e.preventDefault();
+
+    const { searchPerson } = this.state;
+    try {
+      const { data } = await axios.get("https://ghibliapi.herokuapp.com/people");
+      const personName = data.find((person) => person.name === searchPerson);
+      if(personName === undefined) {
+        this.setState({
+          searchPerson: "",
+          currentPerson: {},
+          personName: "",
+          error: true,
+        });
+      }
+      this.setState({
+        searchPerson: "",
+        currentPerson: personName,
+        personName: personName.name,
+        error: false,
+      });
+    } catch (e) {
+      this.setState({
+        searchPerson: "",
+        currentPerson: {},
+        persontName: "",
+        error: true,
+      });
+    }
+    
+  };
+
+  render() {
+    const { searchPerson, currentPerson, personName, isError } = this.state;
+    return (
+      <div>
+        <h1>Search for a Person</h1>
+        <form onSubmit={this.handleSubmit}>
+          <input
+            value={searchPerson}
+            type="text"
+            placeholder="Find Your Person"
+            onChange={this.handleChange}
+          />
+          <button>Submit</button>
+        </form>
+
+        {isError ? (
+          <h1>Not Found</h1>
+        ) : personName ? (
```

Comment:
I would separate the double ternary statements. It makes it a little hard to read. Something like this:
```
{isError ? (
  <h1>Not Found</h1>
)}: null}

{personName ? ( // Your logic problem above(line 36) I think is causing issues but technically if there is an error then personName should be empty

)}: null}
```